playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -62,6 +64,14 @@
   roles:
   - openshift_facts
   post_tasks:
+
+  # Run the pre-upgrade hook if defined:
+  - debug: msg="Running pre-master upgrade hook {{ openshift_master_upgrade_pre_hook }}"
```

Comment:
Instead of adding debug here, I think just adding `name: Running master pre-upgrade hook` to the below should be fine.